### ISSUE TITLE
Activation for TabNetClassifier

### ISSUE THREAD

**Issue by jeongyoonlee** [ostamand/tensorflow-tabnet#8]
Shouldn't the activation for `self.head` of `TabNetClassifier` be `softmax` for multiclass and `sigmoid` for binary classification instead of `None`?

https://github.com/ostamand/tensorflow-tabnet/blob/c4caa8f48f86d55b54b6816818047f12e9dc6bf6/tabnet/models/classify.py#L63

**Comment by ostamand** [comment]
Good point. It should probably an argument that can be provided. I was using `activation=None` because I usually use the activation in the loss calculation for numeric stability ie `tf.keras.losses.BinaryCrossentropy(from_logits=False, ...)`. But this could be refactored.

**Comment by jeongyoonlee** [comment]
Thanks, @ostamand. That makes sense. Just submitted PR to add an input argument for the output activation in `TabNetClassifier`. Please take a look and feel free to update it.

**Comment by ostamand** [comment]
Sweet! Thanks for the PR. Merged :)
